src/main.rs

```diff
@@ -51,15 +49,14 @@ async fn main() {
 
     let app_state = AppState { database: db_data };
 
+
     let app = Router::new()
-        .route("/:uuid", get(return_config_file))
+        .route("/id/:uuid", get(return_config_file))
```

Comment:
You forgot to also update the redirect paths in `redirect_post_multipart_form()` and `redirect()`. Also I think to keep it a bit more consistent I would probably name the path `tar` instead of `id`.